### DISCUSSION TITLE
Github deployments

### DISCUSSION THREAD

**Discussion by RoryFriday** [vercel/community#3846] [Bug Report]
### Page to Investigate

https://www.githubstatus.com/incidents/f7lz4nc72bvp

### Steps to Reproduce

Reviewing the status of the github webhook issue apparently affecting deployments via github ... 

It appears to be resolved https://www.githubstatus.com/incidents/f7lz4nc72bvp

However, we are still unable to trigger builds from github - if github have reported this sorted over an hour ago, what is the expected ETA for this to be sorted Vercels side?

Also, the workaround for deploying via the CLI doesn't seem to work either ...

**Comment by amyegan** [comment]
Hey @RoryFriday. There was an incident related to GitHub webhooks: https://www.vercel-status.com/incidents/r7md342b3d39 

It was marked resolved just a few minutes ago. Please let me know if you're still having trouble with GitHub deployments.
